docs/current/cookbook/snippets/secrets-doppler/main.go

```diff
@@ -0,0 +1 @@
+// TODO
```

Comment:
@vito @marcosnils could you please help me port the Python/Node example to Go?

Reply:
Moved the Doppler recipes to a new PR https://github.com/dagger/dagger/pull/5080

Reply:
Resolving as it will be addressed by #5080